src/pages/merge/MergeIndividuals.jsx

```diff
@@ -0,0 +1,149 @@
+  const showNicknameInput = nameContexts.includes('Nickname');
+  const showResolveFields =
+    showSexInput || showDefaultNameInput || showNicknameInput;
+
+  const formComplete = isFormComplete(
+    formData,
+    showSexInput,
+    showDefaultNameInput,
+    showNicknameInput,
+  );
+
+  return (
+    <MainColumn
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        maxWidth: 1000,
+      }}
+    >
+      <Grid
+        container
+        direction="column"
+        spacing={2}
+        style={{ padding: '20px 6vw' }}
+      >
+        <Grid item>
+          <Text variant="h4" id="MERGE_INDIVIDUALS" />
+        </Grid>
+        <Grid item>
+          {individualData.map(individual => (
+            <IndividualCard
+              key={individual?.guid}
+              data={individual}
+              mergeConflicts={mergeConflicts}
+            />
+          ))}
+        </Grid>
+        {showResolveFields && (
+          <Grid
+            item
+            style={{ display: 'flex', flexDirection: 'column' }}
+          >
+            <Text variant="h5" id="RESOLVE_FIELDS" />
+            {showDefaultNameInput && (
+              <ResolutionSelector
+                value={formData?.defaultName}
+                onChange={newDefaultName =>
+                  setFormData({
+                    ...formData,
+                    defaultName: newDefaultName,
+                  })
+                }
+                fieldType="defaultName"
+                individualData={individualData}
+              />
+            )}
+            {showNicknameInput && (
+              <ResolutionSelector
+                value={formData?.nickname}
+                onChange={newNickname =>
+                  setFormData({ ...formData, nickname: newNickname })
+                }
+                fieldType="nickname"
+                individualData={individualData}
+              />
+            )}
+            {showSexInput && (
+              <ResolutionSelector
+                value={formData?.sex}
+                onChange={newSex =>
+                  setFormData({ ...formData, sex: newSex })
+                }
+                fieldType="sex"
+                individualData={individualData}
+              />
+            )}
+          </Grid>
+        )}
+        <Grid item style={{ marginTop: 16 }}>
+          <Button
+            display="primary"
+            id="MERGE_INDIVIDUALS"
+            loading={loading}
+            disabled={loading || !formComplete}
+            onClick={() => {
+              const propertyOverrides = derivePropertyOverrides(
+                formData,
+                showSexInput,
+                showDefaultNameInput,
+                showNicknameInput,
+              );
+
+              mergeIndividuals({
+                targetIndividualGuid: individual1Data?.guid,
+                fromIndividualGuids: [individual2Data?.guid],
```

Comment:
What do you think about using these types of identifiers for the local variables?
For example, `targetIndividualData` instead of `individual1Data`.

Reply:
i can do that